pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -46,7 +45,7 @@ const (
 	// Time limits for event loop handlers
 	errorTime        = 3 * time.Minute
 	warningTime      = 40 * time.Second
-	verifierBasePath = "/persist/downloads"
+	verifierBasePath = types.DownloadDirname
```

Comment:
I'd like to have a definition and not strings sprinkled in the source, specifically to make today's commit (change to /persist/vault/verifier/) easier. But the use here will have agentName appended.

Reply:
I wouldn't hold it up for that, but you do understand my concern?

Reply:
Yes, I understand it. But I'm more concerned about the use of the pathnames outside of the downloader and verifier than the package in which they are defined. But getting completely rid of those from types package makes sense.

The really odd part is that volumemgr gets to tell the downloader a full path. It would make sure sense to have the downloader pick the top part of the path. Then we can have both the downloader and verifier enforce that their directories are used.